tests/microbenchmarks.py

```diff
@@ -136,11 +153,14 @@ def yarn(n):
             return foo(bar(baz(buzz(bizz(bozz(yam(yarn(n))))))))
         for i in range(100000):
             main(i)
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_cmp(test_case):
     print("Running", "test_cmp")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils